doc/extensibility/property_value_editors.md

```diff
@@ -1,6 +1,53 @@
 Property value editors
 ======================
 
-`IPropertyPageUIValueEditor`
+**[Item template:](project_item_templates.md)** Property Page Value Editor extension
 
-TODO: add docs here
+In some cases, you may want to expand the set of editors to provide a richer experience. For example, you may want to provide a file picker editor to select a file, or a color editor to pick a color.
+
+This can be done by implementing a custom value editor. For that, you will need to:
+### 1. Export the `IPropertyPageUIValueEditor` interface to provide a custom value editor
```

Comment:
Can we avoid using headings as numbered list elements? It looks like the formatting is screwed up when it is rendered as HTML. If your goal is to keep the numbering right, sometimes experimenting with indentation within your numbered item can get the right match.